spec/lib/parse_spec.rb

```diff
@@ -486,6 +486,101 @@
     )
   end
 
+  it 'correctly finds expressions and function calls in a CREATE INDEX statment' do
+    query = described_class.parse('CREATE INDEX testidx ON test USING btree (a, (b+c), lower(d))')
+    expect(query.warnings).to eq []
+    expect(query.tables).to eq ['test']
+    expect(query.ddl_tables).to eq ['test']
+    expect(query.call_functions).to eq ['lower']
```

Comment:
I wonder if its also worth checking `filter_columns` here? (just to have that be deterministic, I'm actually not sure if it'd return an empty table with a column name for regular elements, or no columns at all?)

Reply:
I added filter_columns to the spec and added support for that. Is that what you meant? Maybe it's just my unfamiliarity with pg_query, but I don't quite follow.